llvm/include/llvm/BinaryFormat/ELFRelocs/AArch64.def

```diff
@@ -61,6 +61,7 @@ ELF_RELOC(R_AARCH64_LD64_GOT_LO12_NC,                0x138)
 ELF_RELOC(R_AARCH64_LD64_GOTPAGE_LO15,               0x139)
 ELF_RELOC(R_AARCH64_PLT32,                           0x13a)
 ELF_RELOC(R_AARCH64_GOTPCREL32,                      0x13b)
+ELF_RELOC(R_AARCH64_PATCHINST,                       0x13c)
```

Comment:
Also add a test to tools/llvm-readobj/ELF/reloc-types-aarch64.test

Reply:
Done